### ISSUE TITLE
YAML missing controls AU-13 through AU-16

### ISSUE THREAD

**Issue by m0an0m** [opencontrol/NIST-800-53-Standards#5]
I was trying to set up a sample project based on a real world list of requirements, and noticed that one of the specified Audit controls was not in the YAML, namely, AU-14 (1).  It seems the YAML actually ends the AU section with AU-12.

**Comment by jcscottiii** [comment]
Good catch! I can imagine it wasn't included due to it's low priority / unspecified baseline status
Links:
https://nvd.nist.gov/800-53/Rev4/family/Audit%20and%20Accountability
http://nvlpubs.nist.gov/nistpubs/SpecialPublications/NIST.SP.800-53r4.pdf

Feel free to open a PR to add it!
